rules/Multifactor-Google-Authenticator-Do-Not-Rename.js

```diff
@@ -16,19 +18,17 @@ function (user, context, callback) {
         return callback(null, user, context);
     }
 
-    getUserProfile(user)
-        .then(function (user_profile) {
-            if (!loggedInRecently(user_profile)) {
-                enableMFA(context);
-            }
-
-            return callback(null, user, context);
-        })
-        .catch(function(err) {
-            enableMFA(context);
-            return callback(null, user, context);
-        });
-
+    // Skip MFA if user's IP is on a corporate network
+    // configuration.CORPORATE_NETWORK_CIDRS is a space separated list of IP address ranges (CIDR notation)
```

Comment:
The first sentence of this comment says what the code does, and doesn't add any information, so I'd remove it.